lib/manager/components/CreateFeedSource.js

```diff
@@ -36,13 +37,29 @@ type Validation = {
 }
 
 type State = {
+  loading?: boolean,
   model: NewFeed,
   validation: Validation
 }
 
 export default class CreateFeedSource extends Component<Props, State> {
   componentWillMount () {
     this._initializeState(this.props)
+    window.addEventListener('keydown', this._handleKeyDown)
+  }
+
+  componentWillUnmount () {
+    window.removeEventListener('keydown', this._handleKeyDown)
+  }
+
+  _handleKeyDown = (e: SyntheticKeyboardEvent<HTMLInputElement>) => {
+    switch( e.keyCode ) {
+      case 13: // ENTER
+        this._onSave()
```

Comment:
There needs to be some kind of validation check done before calling `_onSave`.  It's possible to create a feed source without a name with this change.